src/router/routes.ts

```diff
@@ -35,7 +35,7 @@ const routes: Array<RouteConfig> = [
     props: true,
     meta: {
       requiresAuth: true,
-      allowedRoles: [Role.FAS_VIEW]
+      allowedRoles: [Role.FAS_VIEW, Role.FAS_CORRECTION]
```

Comment:
I don't think this is necessary?

Reply:
I have  $route.meta.allowedRoles.includes(Role.FAS_CORRECTION) for roles' validation, so the Adjust Routing Slip button only visible to users with fas_correction role. if it's unnecessary, I'll remove fas_correction for both lines.

Reply:
This route specifically is just for viewing a routing slip. We want to tie this to the permissions FAS_VIEW only.